newsroom/agenda/agenda.py

```diff
@@ -207,6 +206,35 @@ def _agenda_query():
     }
 
 
+def get_utcnow():
+    'added for unit tests'
+    return datetime.utcnow()
+
+
+def today(time, offset):
```

Comment:
imo would be good to put those date related functions into some date_utils module, so we can reuse it and it's not agenda related really